### PR TITLE
Debug update

### PR DESCRIPTION
**Pull request by amyecampbell**
The following pull request adds, fixes, and/or modifies:

1. Small adjustments to fix bugs
2. Removal of two low-correlated TCGA samples (observed both by us and L. Waldron)
3. Removal of doppelgangR installation from INSTALL.R (successfully installed using environment.yml, testing that it works now)

I performed the following prior to filing this pull request:

- [ ] Tested that my change does not break the analysis pipeline
- [ ] Ran a linter through my code
- [ ] Update environment.yml if my code introduces new packages.

Lastly, I will assign individuals to review my code and be proud of making the pull request!

### CODE REVIEW

File: 1.DataInclusion/Scripts/A.getInclusion.R [hunk 1]
```diff
@@ -27,6 +26,14 @@ source("1.DataInclusion/Scripts/Functions/Inclusion_functions.R")
 ####################################
 # Load Constants
 ####################################
+args <- commandArgs(trailingOnly=TRUE)
```
Comment (gwaygenomics) [inline 1.DataInclusion/Scripts/A.getInclusion.R]: I think transitioning commandArgs to [optparse](https://cran.r-project.org/web/packages/optparse/index.html) will be preferable. See [here](https://github.com/greenelab/tad_pathways_pipeline/blob/master/scripts/build_snp_list.R#L29) for an example. 

Optparse makes it clear which variables are being passed, and also will deal with defaults thus removing the need for lines 31-35

File: 1.DataInclusion/Scripts/Functions/LoadOVCA_Data.R [hunk 2]
```diff
@@ -96,11 +105,11 @@ LoadOVCA_Data <- function(datasets,
     
     # Assign the list elements according to the subset of choice
     if (goodsample_subset_dir != "None" & genelist_subset != "None") {
-      ExpData[[eset_exprs]] <- dta[subset[, 1], goodSamples]
+      ExpData[[eset_exprs]] <- dta[subset[, 1], colnames(dta) %in% goodSamples]
```
Comment (gwaygenomics) [inline 1.DataInclusion/Scripts/Functions/LoadOVCA_Data.R]: i think updating `goodSamples` and maintaining the previous subseting step (line 99) would be easier to read. (i mean, update with something like: `goodSamples <- goodSamples[colnames(dta) %in% goodSamples]`

Comment (gwaygenomics) [inline 1.DataInclusion/Scripts/Functions/LoadOVCA_Data.R]: Goodsamples here too

Comment (amyecampbell) [inline 1.DataInclusion/Scripts/Functions/LoadOVCA_Data.R]: ohh i see

**Pull request merged**